### ISSUE TITLE
Change names of metric coefficients

### ISSUE THREAD

**Issue by JensMadsen** [boutproject/BOUT-dev#206]
Very low priority. ....

Why not name metric coefficients e.g. gxx instead of g11? Coordinates are everywhere called (x,y,z) and $gxx = \nabla x \cdot \nabla x$

**Comment by bendudson** [comment]
Agreed. No idea why I chose g11 rather than the more sensible gxx, and now have deep regret. Sorry.

**Comment by JensMadsen** [comment]
You are forgiven:-D No big problem. I just have dictionary in my python scripts:-D

**Comment by bendudson** [comment]
@ZedThree if the Coordinates interface is being changed (and it should) then changing e.g.`g11` to `gxx` is long overdue.
